dev-lisp/ecl/ecl-24.5.10.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="Embeddable Common-Lisp interpreter"
+DESCRIPTION="ECL (Embeddable Common-Lisp) is an interpreter of the \
+Common-Lisp language as described in the X3J13 Ansi specification, featuring \
+CLOS (Common-Lisp Object System), conditions, loops, etc, plus a translator \
+to C, which can produce standalone executables."
+HOMEPAGE="https://www.ecl.org"
+COPYRIGHT="2015 Daniel Kochmański
+	2000 Juan Jose Garcia Ripoll
+	1990, 1991, 1993 Giuseppe Attardi
+	1984 Taiichi Yuasa and Masami Hagiya"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://common-lisp.net/project/ecl/static/files/release/ecl-$portVersion.tgz"
+CHECKSUM_SHA256="e4ea65bb1861e0e495386bfa8bc673bd014e96d3cf9d91e9038f91435cbe622b"
+PATCHES="ecl-24.5.10.patchset"
```

Comment:
```suggestion
PATCHES="ecl-$portVersion.patchset"
```

Reply:
updated and removed previous version along with attempt to split base and devel packages

Reply:
Will run some checks, thanks!